test/e2e/scenario/recorder.scenario.ts

```diff
@@ -654,17 +694,305 @@ describe('recorder', () => {
         expect(segment.records.slice(3).every((record) => record.type !== RecordType.FullSnapshot)).toBe(true)
       })
   })
+
+  describe('layout viewport properties - ', () => {
+    createTest('getWindowWidth/Height reports using layout viewport dimensions')
+      .withRum({ enableExperimentalFeatures: ['visualviewport'] })
+      .withRumInit(initRumAndStartRecording)
+      .withSetup(bundleSetup)
+      .withBody(html`${VIEWPORT_META_TAGS}`)
+      .run(async ({ events }) => {
+        /**
+         * InnerWidth/Height on some devices/browsers are changed by pinch zoom
+         * We need to ensure that our measurements are not affected by pinch zoom
+         */
+        if (isGestureUnsupported()) {
+          return // No Fallback test
+        }
+        await resetViewport()
+
+        const { innerWidth, innerHeight } = (await browserExecute(() => ({
+          innerWidth: window.innerWidth,
+          innerHeight: window.innerHeight,
+        }))) as { innerWidth: number; innerHeight: number }
```

Comment:
https://webdriver.io/docs/api/browser/getWindowSize ?

Reply:
The underlying JSONWP spec wasn't clear what is being returned here, I suspect it is outerWidth/height- not sure if scrollbar width is included for all device browsers. 

While the code is less clean, it's perhaps more accurate to directly test `innerWidth`'s relationship with `visualViewport.scale * visualViewport.width` to be sure they are the same (which is done throughout this file)

Reply:
did you test it?